File: project_templates/server_openapi_generator/java/Handler.java
package {{#if project_info.package}}{{ project_info.package }}.{{/if}}handlers;

import io.vertx.core.Handler;
import io.vertx.ext.web.api.RequestParameters;
import io.vertx.ext.web.RoutingContext;

public class {{ operation.class_name }} implements Handler<RoutingContext> {

    public {{ operation.class_name }}(){

    }

    @Override
    public void handle(RoutingContext routingContext) {
        {{#if renderParams}}RequestParameters params = routingContext.get("parsedParameters");
        {{/if}}// Handle {{ operation.operationId }}
        routingContext.response().setStatusCode(501).setStatusMessage("Not Implemented").end();
    }

}

File: project_templates/server_openapi_generator/kotlin/Handler.kt
package {{#if project_info.package}}{{ project_info.package }}.{{/if}}handlers;

import io.vertx.core.Handler
import io.vertx.ext.web.RequestParameters
import io.vertx.ext.web.RoutingContext

class {{ operation.class_name }} : Handler<RoutingContext> {
    override fun handle(routingContext: RoutingContext) {
        {{#if renderParams}}var params: RequestParameters = routingContext.get("parsedParameters")
        {{/if}}// Handle {{ operation.operationId }}
        routingContext.response().setStatusCode(501).setStatusMessage("Not Implemented").end()
    }
}